### ISSUE TITLE
Do I need to change the learning rate when my training loss aver loop ?

### ISSUE THREAD

**Issue by kamelbouyacoub** [thtrieu/darkflow#118]
Hello 

Currently, I am training my model and, I remark after 1000 steps, the aver  loss is like in a loop, it's down to 0.22 after some steps it's increase to 0.5, after that it's deacrese to 0.2, it loop between 0.2 and 0.5, do I have to change the learning rate ?

Please if anyone have some idea, please advice me 

Thanks

**Comment by thtrieu** [comment]
When loss is oscillating, it might be reasonable to decrease learning rate. However training a deep net is a big topic and won't be discussed further here.
